app/config.js

```diff
@@ -29,6 +29,9 @@ module.exports = {
   cookieText: 'GOV.UK uses cookies to make the site simpler. <a href="#">Find out more about cookies</a>',
 
   // Enable or disable Browser Sync
-  useBrowserSync: 'true'
+  useBrowserSync: 'true',
+
+  // Set a value (in milliseconds) for the max age of all static assets in cache
+  assetsMaxAge: 0
```

Comment:
As discussed on Slack, it might be a more consistent approach to have `useCache`, with that only happening in `production`. Or not have a config at all and just cache on production.